controller/controller.php

```diff
@@ -6,11 +6,18 @@
 use PHPMailer\PHPMailer\OAuth;
 // Alias the League Google OAuth2 provider class
```

Comment:
No comments between uses normally, it doesn't bring much more details

Reply:
Copy that !